rules/falco_rules.yaml

```diff
@@ -1266,7 +1266,9 @@
 
 - rule: Write below etc
   desc: an attempt to write to any file below /etc
-  condition: write_etc_common
+  condition: >
+    write_etc_common
+    and not exe_running_docker_save
```

Comment:
@JPLachance please add `exe_running_docker_save` inside `write_etc_common`

Reply:
I can, but I'll open two PRs for the "Write below etc" rule really soon and the condition will become bigger.

I like it when the condition looks like:
```YAML
condition: >
  something and something_else or this
  and not a_condition_making_it_legit
  and not another_condition_making_it_legit
```

What do you think?

Reply:
`write_etc_common` actually looks like the way you suggested.

Reply:
Ho! Okay, sorry, I miss-read your initial comment. On it!